tripleblindmarket/covid-safe-paths
# Patch for android restart bug in react-native-background-geolocation

## Description

react-native-background-geolocation bug fix that described in #523 

## How to test

Restart your device. You should see the SafePaths notification in drawer within a minute or two, indicating that react-native-background-geolocation is pulling location again.

Fixed the issue using patch-package. The patch show many lines changed due to jettifier, but the patch itself was a single line (L248)
